spec/features/user_views_rotas_spec.rb

```diff
@@ -0,0 +1,91 @@
+require "rails_helper"
+
+RSpec.feature "User views rota" do
+  background { set_data_api_to FakeDataApis::FullDataStore }
+
+  scenario "filtering the rota by date" do
+    procurement_area = create(
+      :procurement_area,
+      name: "Gotham",
+      memberships: [
+        {
+          uid: "e6256f3b-3920-4e5c-a8e1-5b6277985ca1",
+          name: "Brighton Custody Suite",
+          type: "custody_suite",
+        },
+        {
+          uid: "32252f6a-a6a5-4f52-8ede-58d6127ba231",
+          name: "Guilded Groom & Groom",
+          type: "law_firm",
+        }
+      ]
+    )
+
+    shift = create(:shift,
+                   name: "Normal shift",
+                   location_uid: "e6256f3b-3920-4e5c-a8e1-5b6277985ca1")
+
+    create(:rota_slot,
+           procurement_area: procurement_area,
+           starting_time: Time.parse("01/01/2015 09:00"),
+           shift: shift,
+           organisation_uid: "32252f6a-a6a5-4f52-8ede-58d6127ba231"
+          )
+
+    create(:rota_slot,
+           procurement_area: procurement_area,
+           starting_time: Time.parse("01/02/2015 09:00"),
+           shift: shift,
+           organisation_uid: "32252f6a-a6a5-4f52-8ede-58d6127ba231"
+          )
+
+    create(:rota_slot,
+           procurement_area: procurement_area,
+           starting_time: Time.parse("01/12/2014 09:00"),
+           shift: shift,
+           organisation_uid: "32252f6a-a6a5-4f52-8ede-58d6127ba231"
+          )
+
+    admin_user = create :admin_user
+
+    login_with admin_user
+    visit procurement_area_path procurement_area
+    click_link "Manage rotas"
+
+    expect(page).to have_content("Rotas for Gotham")
+
+    expect(page).to have_content("Monday, 1 Dec 2014")
+    expect(page).to have_content("Thursday, 1 Jan 2015")
+    expect(page).to have_content("Sunday, 1 Feb 2015")
+
+    select "2015", from: "rota_filter[starting_date(1i)]"
+    select "January", from: "rota_filter[starting_date(2i)]"
+    select "1", from: "rota_filter[starting_date(3i)]"
```

Comment:
I think the interaction with the date fields is pretty difficult to read. Could we extract it into a method: à la http://stackoverflow.com/questions/6729786/how-to-select-date-from-a-select-box-using-capybara-in-rails-3